src/ontology/efo-edit.owl

```diff
@@ -252181,6 +252181,7 @@ AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_1001491 "ICD10:O04")
 AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_1001491 "ICD10:O06")
 AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_1001491 "MedDRA:10000210")
 AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_1001491 "Wikipedia:Abortion")
+AnnotationAssertion(Annotation(skos:narrowMatch "") oboInOwl:hasDbXref efo:EFO_1001491 "MESH:D000028")
```

Comment:
[abortion](http://www.ebi.ac.uk/efo/EFO_1001491) is defined in EFO as "the ending of pregnancy by removing a fetus or embryo before it can survive outside the uterus." Looking at its subclasses and their definition, it appears to cover both induced abortions and miscarriage.

This PR adds a mesh xref to [Abortion, Induced](https://id.nlm.nih.gov/mesh/D000028.html), which is narrower because it excludes miscarriage. Therefore I used the `skos:narrowMatch` axiom.